Patches/SQLite3/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+project ( SQLite3 )
+cmake_minimum_required(VERSION 2.8)
```

Comment:
This seems likely to produce warnings. Maybe we should require a newer version, since fletch itself already requires at least 3.3?